File: Chapter_2_client/src/main/java/org/lovedev/client/MainActivity.java
package org.lovedev.client;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void navToBookManager(View view) {
        startActivity(new Intent(this, BookManagerActivity.class));
    }

    public void navToBinderPool(View view) {
        startActivity(new Intent(this, BinderPoolActivity.class));
    }
}

File: chapter_10/src/main/java/org/lovedev/chapter_10/MainActivity.kt
package org.lovedev.chapter_10

import android.os.Bundle
import android.os.SystemClock
import android.support.v7.app.AppCompatActivity
import android.view.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun click(view: View) {
        SystemClock.sleep(5100)
    }
}
